testplan/common/utils/networking.py

```diff
@@ -8,10 +8,65 @@
     absolute_import,
 )
 import socket
+import ipaddress
```

Comment:
Looks like `ipaddress` was only [added in Python 3.3](https://docs.python.org/3.5/library/ipaddress.html?highlight=New%20in%20version%203.3.). Perhaps you could add this [ipaddress](https://pypi.org/project/ipaddress/) as a [conditional dependency](https://stackoverflow.com/a/35614580)?

```txt
ipaddress==1.0.23 ; python_version < '3.3'
```

Reply:
We already have ipaddress as a dependency in setup.py. Good point on making it a conditional dependency though :+1: